src/main/resources/language.yml

```diff
@@ -256,3 +256,9 @@ track-new2: '&a&l |&6&l   Swing the stick to track players!                &a&l
 lb-blank-space: ''
 lb-combined-separator: ' : '
 lb-missing-player: '<none>'
+
+#leavegame item name
+leave_game: 'Leave Game'
+
+#Force start item name
+force-start: 'force start game'
```

Comment:
Rather than notes, I would just make the keys more self explanatory
```suggestion

# LEAVE/START GAME ITEMS
leave-game-item-name: 'Leave Game'
force-start-item-name: 'Force Start Game'
```